src/model.rs

```diff
@@ -8,6 +11,38 @@ pub struct Contexts {
     pub name: String,
 }
 
+#[derive(Default, Clone, PartialEq, Debug, Serialize, Deserialize)]
+pub struct Clusters {
+    #[serde(default)]
+    pub cluster: Cluster,
+    #[serde(default)]
+    pub name: String,
+}
+#[derive(Default, Clone, PartialEq, Debug, Serialize, Deserialize)]
+pub struct Cluster {
+    #[serde(rename = "certificate-authority-data")]
+    pub certificate_authority_data: String,
+    pub server: String,
+}
+
+#[derive(Default, Clone, PartialEq, Debug, Serialize, Deserialize)]
+pub struct Users {
+    #[serde(default)]
+    pub user: HashMap<String, Value>,
```

Comment:
Using hashmap here because we aren't interested in the structure and they can vary from providers (AWS,k3s,GKE)